### DISCUSSION TITLE
Welcome to CronjobManager Discussions!

### DISCUSSION THREAD

**Discussion by Ethan3600** [Ethan3600/EthanYehuda_CronjobManager#163] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by janbiesiada** [comment]
Please share info about extension compatibility with 
magento/product-enterprise-edition:2.4.7
